js/server/tests/resilience/moving-shards-cluster.js

```diff
@@ -132,6 +132,37 @@ function MovingShardsSuite () {
 
 
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief get cleaned out servers
+////////////////////////////////////////////////////////////////////////////////
+
+  function getTransientValue() {
```

Comment:
This function is not used anywhere, do we need it? If not, I suggest removing it.